app.js

```diff
@@ -2,72 +2,95 @@
 
-    // increment the head change count state
-    
-    // update the dom for the head (use style.backgroundImage on the headEl div instead of trying to set the .src -- it's NOT an img tag!)
+    headCount++;
 
-    // update the stats to show the new count (call displayStats() to do this work)
+    headEl.style.backgroundImage = `url(./assets/${headDropdown.value}-head.png)`;
+
+    displayStats ();
 });
 
 
 middleDropdown.addEventListener('change', () => {
-    // get the value of the middle dropdown
 
-    // increment the middle change count state
-    
-    // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
+    const value = middleDropdown.value;
+
+    middleCount++;
+
+    middleEl.style.backgroundImage = `url(./assets/${middleDropdown.value}-middle.png)`;
 
-    // update the stats to show the new count (call displayStats() to do this work)
+    displayStats ();
 });
 
 
 bottomDropdown.addEventListener('change', () => {
-    // get the value of the bottom dropdown
-
-    // increment the bottom change count state
     
-    // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
+    const value = bottomDropdown.value;
+   
+    bottomCount++;
+
+    bottomEl.style.backgroundImage = `url(./assets/${bottomDropdown.value}-pants.png)`;
 
-    // update the stats to show the new count (call displayStats() to do this work)
+    displayStats ();
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
-    // push the new catchphrase to the catchphrase array in state
+    const newCatchphrase = catchphraseInput.value;
+
+    catchphrases.push(newCatchphrase);
+
+    catchphraseInput.value = '';
 
-    // clear out the form input's value so it's empty to the user
-   
-    // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
 
+    displayCatchphrases();
 });
 
 function displayStats() {
-    // text content of the reportEl to tell the user how many times they've changed each piece of the state
+    const statsString = createStatsString(headCount, middleCount, bottomCount);
+    
+    reportEl.textContent = statsString;
 }
 
 function displayCatchphrases() {
-    // clear out the DOM for the currently displayed catchphrases
 
-    // loop through each catchphrase in state
-   
-    // and for each catchphrase
-    
-    // create an HTML element with the catchphrase as its text content
+    catchphrasesEl.textContent = '';
+
+    for (let catchphrase of catchphrases) {
+        const p = document.createElement('p');
     
-    // and append that HTML element to the cleared-out DOM
+
+        catchphrasesEl.classList.add('catchphrase');
+        catchphrasesEl.textContent = catchphrases;
+
+        catchphrasesEl.append('p');
```

Comment:
I think you want to append the variable p you defined on line 83 above and not a string 'p' here.